torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -60,11 +65,13 @@
     DistTestCases
 )
 from torch.testing._internal.common_utils import (
+    instantiate_parametrized_tests,
     IS_MACOS,
     IS_WINDOWS,
     FILE_SCHEMA,
     IS_FBCODE,
     NO_MULTIPROCESSING_SPAWN,
+    parametrize,
```

Comment:
Or maybe not? I am not a linguistic expert so not sure whether it should be called parametrize or parameterize :)